%YAML:1.0

width: 640
height: 480
cam0:
  distortion_coeffs: [0.037777619230207594, -0.03281359663234099, 0.0028477403978813295,
    -0.004394608503288771]
  intrinsics: [545.4944662280602, 517.6326626919627, 294.9981214394193, 274.13952920501436]
  rostopic: /cam1/image_raw
cam1:
  distortion_coeffs: [0.028278187782053998, -0.02338648164212651, 0.0023209256543983035,
    -0.005064146236225701]
  intrinsics: [456.26558990143167, 457.8208737395829, 294.9080809113519, 266.7875792958226]
  rostopic: /cam2/image_raw
T10: !!opencv-matrix
  rows: 4
  cols: 4
  dt: d
  data: [0.9999453505599502, 0.0008016449599122864, 0.010423687394213593, -0.0001468997832048657,
        -0.0007877973365491005, 0.9999988018893853, -0.001332514595063151, -0.01766637332683568,
        -0.010424743109092246, 0.0013242300207204935, 0.999944784048578, -0.0007158494591820072,
        0.0, 0.0, 0.0, 1.0]
